test/units/inventory/test_inventory.py

```diff
@@ -115,3 +114,55 @@ def test_expand_hostname_range(self):
         for e in self.ranges_to_expand:
             r = self.ranges_to_expand[e]
             self.assertEqual(r, expand_hostname_range(e))
+
+
+class InventoryDefaultGroup(unittest.TestCase):
+
+    def setUp(self):
+        v = VariableManager()
+
+        fake_loader = DictDataLoader({
+            'hosts': self.inventory_content
+        })
+
+        with patch.object(Inventory, 'basedir') as mock_basedir:
+            mock_basedir.return_value = './'
+            self.inventory = Inventory(loader=fake_loader, variable_manager=v, host_list='hosts')
+
+    def test_default_groups(self):
+        self.assertIn('all', self.inventory.groups)
+        self.assertIn('ungrouped', self.inventory.groups)
+        all_hosts = set([host.name for host in self.inventory.groups['all'].get_hosts()])
+        self.assertEqual({'host1', 'host2', 'host3', 'host4', 'host5'}, all_hosts)
```

Comment:
CI failure due to python 2.6 syntax error:

```
2017-02-21 17:30:19 SyntaxError: ('invalid syntax', ('./test/units/inventory/test_inventory.py', 136, 34, "        self.assertEqual({'host1', 'host2', 'host3', 'host4', 'host5'}, all_hosts)\n"))
```

Set literal expressions are not supported on python 2.6.